xtable-core/src/main/java/org/apache/xtable/delta/DeltaConversionTarget.java

```diff
@@ -66,9 +66,9 @@
 import org.apache.xtable.spi.sync.ConversionTarget;
 
 public class DeltaConversionTarget implements ConversionTarget {
-  private static final String MIN_READER_VERSION = String.valueOf(1);
+  private static final String MIN_READER_VERSION = String.valueOf(3);
```

Comment:
Handle this gracefully by calling the function which upgrades version in delta codebase, this was the pending comment from the previous PR that wasn't addressed.

https://github.com/apache/incubator-xtable/pull/428

Reply:
Is this change related to `timestamp_ntz`? Increasing the `min_reader` version could break certain consumers using old libraries. If it's unrelated, could we create a separate issue for this and perhaps make it configurable?

Reply:
Delta does the automatic version upgrade today based on the schema of the table and other properties, if we call the same function when initializing the delta table it shouldn't break anything. 
 
https://docs.delta.io/2.0.0/versioning.html 
`When creating a table, Delta Lake chooses the minimum required protocol version based on table characteristics such as the schema or table properties`

Reply:
Thanks for the pointers. 
However, I believe this change will force an upgrade to the `min_reader` version, even if none of the table features require it. I support updating the version, but lets not mix it in this PR unless it is needed for timezone handling.

Reply:
Can we start with the existing min reader/writer on the first commit and then parse back the versions for the subsequent commits? That is what I was asking for on the other draft for this feature.

Reply:
Handled backwards compatibility